src/_posts/2020-12-04-software-apprenticeship-the-heart-of-codurance.md

```diff
@@ -39,15 +39,15 @@ As the above quote demonstrates, the concept of an apprenticeship dovetails nice
 
-#### The Codurance Academy
+##The Codurance Academy
 
 As mentioned in the introduction of this article, the apprenticeship program at Codurance has evolved over time along with the Organisation. The founders of Codurance placed the apprenticeship program at the core of the Organisation from the very outset.  The initial setup of the program saw apprentices assigned a mentor on a one-to-one basis. The focus was on passing on the core Software Craftsmanship principles that are at the very heart of Codurance and in turn the London Software Craftsmanship Community (LSCC). These include Agile technical practices such as Test Driven Development but also wider software design concepts and known best practices, to instill the pragmatic value-focussed mindset and interpersonal skills required to be successful in a [software development team](https://www.codurance.com/publications/2020/09/24/what-are-the-skills-that-each-member-of-your-team-must-have).  
 
 Whilst this enabled our apprenticeships to benefit from our more experienced craftspeople there were some shortcomings from this setup. The core skills were being passed on but there was no consistency in the approach followed by mentors. As a result we saw mentors focussing more on those areas where they had a particular strength or interest and less in areas where they did not. This meant that whilst the basic skills were being covered, apprentices were perhaps coming out the program having gaps in certain areas. At that time, the apprenticeships did not have a formal curriculum, it was left to the mentor as to what should be taught and indeed how that should be taught. So apprentices were experiencing different programmes depending on the mentor they had been assigned. 
 
 At Codurance, we are very strong believers in continual learning and development and we wanted to apply this to our academy program. We noticed that the program lacked some consistency. One of the reasons we value the academy program so much is that we feel it brings a certain level of professional standard when our people work with our clients. In order to fully achieve this, we felt that we needed to make some adjustments to the program to make sure that all of our apprentices were exposed to the same learning program and that at the end of the program, we could be confident that they were all able to operate at the same level that we, as Codurance pride ourselves in.  There were a number of key changes that we introduced, which we believe have significantly improved the program : 
 
-**Formal Programme**
```

Comment:
The rest of the bolds were turned into third-level headers but this one was turned into a second-level header, is that intentional?

Reply:
no... not enough hashes. It's a bit cold today in Barcelona and my fingers are freezing :)